site: Add julia manifest

juliaogris opened the pull request:
Copy most of camh's setup from `manifests/camh` to `manifests/julia`.
Add julia manifest for Julia's domain (jul.run) and email (me@juliaogris.com). 
Remove everything related to metallb - I couldn't get metallb and virtual
IPs to work on my Google WiFi home network.

Add the Traefik dashboard port to the NodePort Service so it can be
reached without a loadbalancer in `manifest/traefik`

camh- commented on manifests/julia/traefik-services.yaml:
```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: traefik-external-np
```
This NodePort Service has been moved to `manifests/traefik` so it is not needed here any more. There was nothing site-specific about it, so I moved it.

juliaogris replied:
ok, removed entire file as discussed.

camh- commented on manifests/julia/traefik-services.yaml:
```diff
@@ -0,0 +1,37 @@
+    app: traefik
+spec:
+  type: LoadBalancer
+  loadBalancerIP: 192.168.86.200
```
my `traefik-services.yaml` file (camh) has two extra load balancers. the internal one (.201) probably wont work for you without your own internal DNS, but the traefik dashboard one (.202) should I think

juliaogris replied:
same here.

camh- commented on manifests/julia/cert-manager-issuers.yaml:
```diff
@@ -0,0 +1,38 @@
+spec:
+  acme:
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    email: me@juliaogris.com
```
haven't seen you use that address before. hello you!

juliaogris replied:
🌊

juliaogris committed:
traefik: Add nodePort for traefik dashboard

Add the traefik dashboard port to the NodePort Service so it can be
reached without a loadbalancer.

juliaogris committed:
site: Add julia manifest

Add julia manifest for Julia's email (me@juliaogris.com) and domain
(jul.run). Copy most of camh's setup in manifests/camh, but remove
everything related to metallb - I couldn't get metallb and virtual
IPs to work on my Google WiFi home network.

camh- reviewed (commented):
LGTM, but the PR description needs to be updated for the changes you have made, noting that you do not have any load balancer configs for your site. I wonder how we can make the installation of metallb conditional? I'll think about it.

camh- reviewed (approved):
LGTM :leafy_green:
